### DISCUSSION TITLE
Does gatsby-plugin-chakra-ui support chakra-ui latest version? (@chakra-ui/core@next )

### DISCUSSION THREAD

**Discussion by shoxton** [chakra-ui/chakra-ui#2378] [Q&A] [answered]
Are the latest version features available to use with [`gatsby-plugin-chakra-ui`](https://www.gatsbyjs.com/plugins/gatsby-plugin-chakra-ui/)?
If so, could this be more clarified at the [plugin page](https://www.gatsbyjs.com/plugins/gatsby-plugin-chakra-ui/)?

**Comment by ljosberinn** [comment, marked as answer]
I'm unaware how we can change that text but if we can, we will try to adjust once v1 is out.

**Reply by shoxton** [reply]
Thanks for replying.
Is there a release date?

**Reply by ljosberinn** [reply]
We don't have a specific date but the current `rc` is hopefully the last :)

**Reply by Kevpedia** [reply]
there is also https://www.gatsbyjs.com/plugins/@chakra-ui/gatsby-plugin/ which is referenced by Gatsby's own docs: https://www.gatsbyjs.com/docs/how-to/styling/other-css-frameworks/#chakra-ui

I'm a bit confused about which is the right one? The readme on both are also not very helpful in understanding how to adjust the theme in the Gatsby environment

**Reply by jmiazga** [reply]
https://www.gatsbyjs.com/plugins/@chakra-ui/gatsby-plugin/ is for Chakra v1
